Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -127,6 +127,10 @@ public Q_SLOTS:
   Viewer_impl* d;
   double prev_radius;
 
+public:
+  bool isRecent() const;
+  QOpenGLFunctions_4_3_Compatibility* recentFunctions();
```

Comment:
The name `recentFunctions` is badly chosen. Imagine you read that doc in 10 years... It should be, for example, `openGLFunctions_4_3`…